package.json

```diff
@@ -10,13 +10,16 @@
     "type": "git",
     "url": "https://github.com/blackboard/react-dnd-ax.git"
   },
+  "publishConfig": {
+    "registry": "http://npm.bbpd.io/repository/bb-npm-private/"
```

Comment:
Same here. I think we should remove this publishConfig.

Reply:
Agreed.